core/src/main/resources/lib/form/textarea.jelly

```diff
@@ -98,12 +98,12 @@ THE SOFTWARE.
         <j:if test="${value != null &amp;&amp; !empty(value.toString()) &amp;&amp; (value.toString().codePointAt(0) == 10 || value.toString().codePointAt(0) == 13)}"><j:whitespace>&#10;</j:whitespace></j:if>
         <st:out value="${value}" />
       </textarea>
+    <j:if test="${customizedFields != null and attrs.field != null and value != default}">
+      <j:mute>${customizedFields.add(name)}</j:mute>
+    </j:if>
+    <!-- resize handle -->
+    <div class="textarea-handle"/>
   </f:possibleReadOnlyField>
-  <j:if test="${customizedFields != null and attrs.field != null and value != default}">
-    <j:mute>${customizedFields.add(name)}</j:mute>
-  </j:if>
-  <!-- resize handle -->
-  <div class="textarea-handle"/>
```

Comment:
I think this one deserves an `lts-candidate` maybe?

Reply:
Separate PR/bug please, then

Reply:
it's a minor fix that looked especially bad here without the fix, can create another PR if needed but not sure if worth it.

Reply:
Just look at any ExtendedRead job configuration with a shell build step.

Reply:
https://github.com/jenkinsci/jenkins/pull/4536